src/APICore.ts

```diff
@@ -85,4 +87,19 @@ export class APICore {
   private isThrottled(res: Response): boolean {
     return res.status === 429;
   }
+
+  private validateRetryOptions() {
+    let waitTime = this.options.retryAfter;
+    let totalMs = this.options.retryAfter;
+
+    // Compute the theoretical maximum execution time to ensure it's less than 1.5 hours.
+    for (let i = 1; i < this.options.maxRetries; i++) {
+      waitTime *= this.options.timeMultiple;
+      totalMs += waitTime;
+    }
+
+    if (totalMs > 5400000) {
+      throw new Error('Maximum time exceeded');
```

Comment:
Do you think a specific error can be used instead (e.g. `throw new ExceededTimeError('...')`)?
This could help with the error handling. Since the CLI leverages the SDK, having a dedicated error class would help the CLI including additional information and maybe links when the error occurs (instead of simply returning `Maximum time exceeded`)

Reply:
We actually ended up removing this limit/error altogether, so this is no longer a problem hehe